main.go

```diff
@@ -181,8 +193,8 @@ func main() {
 	logger.Infof(nil, "Waiting for connection and authorization")
 	<-connected
```

Comment:
I've concern that when the entities definition has latency 3s for any reason, and the agent starting for the first time, the agent will try to init the gwClient but the auth or connection is delayed, and since the liveness doesn't wait it will make the pod as bad and restart it, without waiting some time before calling the pod.

So I believe adding the below to the probes of the agent deployment, would help.
```
     initialDelaySeconds: 5
```




@iahvector

Reply:
Liveness probe succeeds as soon as the container starts and doesn't wait for anything. Readiness probe is the one that waits, and I think the default values for the probe delays and retries should be enough for this